pkgs/games/xash3d/change-zip-date.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/scripts/waifulib/zip.py b/scripts/waifulib/zip.py
+index 6e903ba7..d47f1754 100644
+--- a/scripts/waifulib/zip.py
++++ b/scripts/waifulib/zip.py
+@@ -25,7 +25,11 @@ class ziparchive(Task.Task):
+ 				arcfile = src.path_from(self.relative_to)
+ 
+ 				Logs.debug('%s: %s <- %s as %s', self.__class__.__name__, outfile, infile, arcfile)
+-				zf.write(infile, arcfile)
```

Comment:
you can usually change `ZipFile` initialization to use `strict_timestamps=False` instead

Reply:
I don't understand. Plus this patch comes from you [here](https://gist.github.com/chayleaf/a8ad389e520e58b885779e051450707e) :p